releases/patch-releases.md

```diff
@@ -57,8 +57,6 @@ releases may also occur in between these.
 
 ### 1.20
 
-Next patch release is **1.20.3**
```

Comment:
This seems like a good change to me given the current workflow, but I think we should look into automatically updating this page in the future. Seems like it could be a pretty easy modification of some existing release tooling 👍

Reply:
I think @cpanato was planning something like that. Let's chat the next time we get together.

Reply:
i was planning/working in the notification system, but we can add a flow to update this page automagically with a prow job. But we will need some manual input to trigger this, maybe we update the schedule yaml file and when we merge that it triggers a prwo job to update the page?